queries/__init__.py

```diff
@@ -397,13 +397,13 @@ def gen_answer(a: str) -> AnswerTuple:
     return dict(answer=a), a, a
 
 
-def query_json_api(url: str) -> Optional[Dict]:
-    """ Request the URL, expecting a JSON response which is
-        parsed and returned as a Python data structure. """
+def query_json_api(url: str, headers: Optional[Dict] = None) -> Optional[Dict]:
+    """Request the URL, expecting a JSON response which is
+    parsed and returned as a Python data structure."""
 
     # Send request
     try:
-        r = requests.get(url)
+        r = requests.get(url, headers=headers)
```

Comment:
Does this work even if `headers` is `None`?

Reply:
Yes.